src/classes/GE_GiftEntryController.cls

```diff
@@ -147,11 +148,11 @@ public with sharing class GE_GiftEntryController {
             upsertDataImport(dataImport);
             return BGE_DataImportBatchEntry_CTRL.runSingleDryRun(dataImport.Id, batchId);
 
-        } catch (Exception ex) {
+        } catch (DmlException ex) {
             // create custom lwc exception and throw it
             String JSONExceptionData = ERR_ExceptionData.createExceptionWrapperJSONString(ex);
 
-            throw returnDmlException(JSONExceptionData);
+//            throw returnDmlException(JSONExceptionData);
```

Comment:
@dfuller114 This is causing a deploy error since the method demands a string type! Do we have a solid understanding of what we want our error handling behavior to be in the case where we land in this catch?

Reply:
@ptewson-sfdo i think that failure might have been due to a leftover artifact that I just removed and pushed up. The deploy should be successful now. Here, we're expecting a DML exception so when it goes into the catch we'll return a `DMLException` data type